src/visit_vtk/offscreen/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ ${VTK_INCLUDE_DIRS}
 
 add_library(visit_vtk_offscreen vtkOffScreenRenderingFactory.C)
 
+if(VISIT_VTK_VERSION VERSION_GREATER_EQUAL "9.1.0")
```

Comment:
Why greater than or equal to 9.1? Do we plan to support versions of VTK that are greater than 9.1?

Reply:
Yes, eventually. If it happens before we remove VTK 8 support, this code block won't need to be modified.
There is a check in the find module for VTK that currently restricts to only 8.1.0 or 9.1.0.